paper-linear-progress.html

```diff
@@ -0,0 +1,897 @@
+<!--
+@license
+Copyright (c) 2015 The Polymer Project Authors. All rights reserved.
+This code may only be used under the BSD style license found at http://polymer.github.io/LICENSE.txt
+The complete set of authors may be found at http://polymer.github.io/AUTHORS.txt
+The complete set of contributors may be found at http://polymer.github.io/CONTRIBUTORS.txt
+Code distributed by Google as part of the polymer project is also
+subject to an additional IP rights grant found at http://polymer.github.io/PATENTS.txt
+-->
+
+<link rel="import" href="../polymer/polymer.html">
+<link rel="import" href="../paper-styles/color.html">
+<link rel="import" href="../neon-animation/neon-animation-runner-behavior.html">
+<link rel="import" href="../neon-animation/animations/transform-animation.html">
+
+<!--
+Material design: [Progress & activity](https://www.google.com/design/spec/components/progress-activity.html)
+
+Linear Progress elements are used to represent an operation that takes a known or unknown (aka
+"indeterminate") amount of time. They can be reversed in order to support RTL layouts and further
+differentiate between different phases of an operation. They can also display a "buffering" state,
+which is useful for operations involving the download of large content (e.g. media).
+
+Example:
+
+    <paper-linear-progress value="10"></paper-linear-progress>
+
+Example:
+
+    <paper-linear-progress buffering value="40" buffer-value="70"></paper-linear-progress>
+
+Example:
+
+    <paper-linear-progress indeterminate reversed></paper-linear-progress>
+
+### Styling the linear progress element:
+
+To change the progress bar color:
+
+    paper-linear-progress {
+       --paper-linear-progress-progress-bar-color: #e91e63;
+    }
+
+Defaults to `--google-blue-500`.
+
+To change the buffer bar color:
+
+    paper-linear-progress {
+      --paper-linear-progress-buffer-bar-color: #f8bbd0;
+    }
+
+Defaults to `--google-blue-100`.
+
+> _NOTE_: The above custom properties are provided by [paper-styles](https://github.com/PolymerElements/paper-styles).
+
+To change the height of the element:
+
+    paper-linear-progress {
+      --paper-linear-progress-height: 2px;
+    }
+
+Defaults to `4px`.
```

Comment:
Please express exposed mixins and custom properties in a markdown table as it was in the original element (https://github.com/PolymerElements/paper-linear-progress/pull/1/files#diff-66187b7889d4c6bd24995a8ee2542e77L65).